config/react-native.js

```diff
@@ -3,11 +3,11 @@
  */
 module.exports = {
   settings: {
-    'import/resolver': {
+    'i/resolver': {
```

Comment:
```suggestion
    'i/resolve': {
```

Settings for `enhanced-resolve`.